app/assets/stylesheets/forms.scss

```diff
@@ -17,3 +20,9 @@ div#votes div {
 div#votes{
   text-align: right;
 }
+div#votes p#voteerror {
+  display: block;
+
+  color: #B10E1E;
```

Comment:
This looks very similar to the `.govuk-error-summary.alert` rule.  Could we re-use this one?